neuraxle/base.py

```diff
@@ -2709,6 +2705,16 @@ def fit(self, data_inputs, expected_outputs=None) -> 'FitCallbackStep':
         fit methods are not implemented
     """
 
+    def _transform_data_container(self, data_container: DataContainer, context: ExecutionContext) -> DataContainer:
+        """
+        Do nothing - return the same data.
```

Comment:
I'm surprised this one wasn't implemented !

Reply:
Not totally sure, but I think I saw that change in one of the old branch that were never merged. Anyway, it's there now :)

Reply:
This should fix weird recursion errors in one of our client's project that was happening under some circonstances when using this mixin. Good!